backend/app/views/spree/admin/product_properties/_product_property_fields.html.erb

```diff
@@ -12,8 +12,11 @@
     <%= f.text_field :value %>
   </td>
   <td class="actions">
-    <% if f.object.persisted? && can?(:destroy, f.object) %>
```

Comment:
Why this `can?` check has been removed?

Reply:
For unifying the both approaches for option values or product properties. You are right, but: Shouldn't these can? constraints all be always implemented on both objects? i would prefer it this way (so can?(:destroy) and can?(:update_positions) in both views..

Reply:
I think that now that we have the `else`, we need to separate the two if conditions, otherwise, when the object is persisted but the user can't destroy the object, the `.spree_remove_fields` versions of the link will be shown. Isn't it?